### DISCUSSION TITLE
Question: is there a way to determine if a job has exhausted all retries

### DISCUSSION THREAD

**Discussion by ebelyaev** [OptimalBits/bull#2078] [Q&A]
## Description

Is there a way to determine if a job has exhausted all retries.

## Bull version

3.22.8

**Comment by manast** [comment]
You could check ```job.attemptsMade >= job.attempts```.

**Reply by ebelyaev** [reply]
Thanks! Looking closer at the API I believe it should be: `job.attemptsMade >= job.opts.attempts`

**Reply by manast** [reply]
yes, sorry I misspelled it.
